### ISSUE TITLE
mlflow logs validation loss as 0

### ISSUE THREAD

**Issue by PeterCalifano** [PeterCalifano/pyTorchAutoForge#2]
Logging system not fully working. Mlflow logs incorrect validation loss value for classification problem.
